### PR TITLE
simulate upload errors using simple filename schema

### PR DESCRIPTION
**Pull request by arturopala**

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.3.13
\ No newline at end of file
+sbt.version=1.6.2
```
Comment (chotai) [inline project/build.properties]: 1.5.8 might be better (preloaded on agents for faster builds)

File: it/services/NotificationSenderISpec.scala [hunk 2]
```diff
@@ -228,6 +228,141 @@ class NotificationSenderISpec
...
+            )
+          )
+          .toString
+        verify(
+          1,
+          postRequestedFor(urlEqualTo("/upscan/callback")).withRequestBody(containing(expectedCallback.toString)))
+      }
+    }
+
+    "initiate a request, upload a file, make a callback, and report the error for a simulated invalid file" in {
```
Comment (chotai) [inline it/services/NotificationSenderISpec.scala:299]: Can we add in a test scenario for "unknown" please? (we have scenarios for rejected, infected, invalid)

Comment (arturopala) [reply it/services/NotificationSenderISpec.scala:299]: done

File: project/build.properties (hunk 1 above)
Comment (arturopala) [reply project/build.properties]: done

**Review by chotai — APPROVED** [review]

**Pull request merged**